assets/migrations/sqlite/005_initialize_schema.sql

```diff
@@ -0,0 +1,63 @@
+-- +goose Up
+CREATE TABLE tuple (
+    store CHAR(26) NOT NULL,
+    object_type VARCHAR(128) NOT NULL,
+    object_id VARCHAR(128) NOT NULL,
+    relation VARCHAR(50) NOT NULL,
+    _user VARCHAR(256) NOT NULL,
+    user_type VARCHAR(7) NOT NULL,
+    ulid CHAR(26) NOT NULL,
+    inserted_at TIMESTAMP NOT NULL,
+    condition_name VARCHAR(256),
+    condition_context LONGBLOB,
+    PRIMARY KEY (store, object_type, object_id, relation, _user)
+);
+
+CREATE UNIQUE INDEX idx_tuple_ulid ON tuple (ulid);
+CREATE INDEX idx_reverse_lookup_user ON tuple (store, object_type, relation, _user);
+
+CREATE TABLE authorization_model (
+    store CHAR(26) NOT NULL,
+    authorization_model_id CHAR(26) NOT NULL,
+    type VARCHAR(256) NOT NULL,
+    type_definition BLOB,
+    schema_version VARCHAR(5) NOT NULL DEFAULT '1.0',
+    serialized_protobuf LONGBLOB,
+    PRIMARY KEY (store, authorization_model_id, type)
+);
```

Comment:
```suggestion
CREATE TABLE authorization_model (
    store CHAR(26) NOT NULL,
    authorization_model_id CHAR(26) NOT NULL,
    schema_version VARCHAR(5) NOT NULL DEFAULT '1.1',
    serialized_protobuf LONGBLOB NOT NULL,
    PRIMARY KEY (store, authorization_model_id)
);
```

Reply:
Right now both of these columns are named in the insert query [here](https://github.com/openfga/openfga/blob/23e48b76f6cf24422c6f3517f5d5f9f7fa216ab8/pkg/storage/sqlcommon/sqlcommon.go#L579) so even though they aren't used we cannot drop them from this table without making some supporting changes.

`type_definition` is nullable so we should be able to just remove it from the insert, but `type` is declared not null so the query is explicitly inserting an empty string to keep MySQL and PostgreSQL happy.

One option may be to write migrations to drop those columns from MySQL and PostgreSQL, but there is also the problem of `type` being part of the primary key. It may not be possible to drop it from the primary key if there are entries in older databases that have duplicate values for `store` and `authorization_model_id`.

Reply:
> Right now both of these columns are named in the insert query [here](https://github.com/openfga/openfga/blob/23e48b76f6cf24422c6f3517f5d5f9f7fa216ab8/pkg/storage/sqlcommon/sqlcommon.go#L579) so even though they aren't used we cannot drop them from this table without making some supporting changes.

I think it's okay to copy-paste this code into the SQLLite implementation and then modify it according to our needs :)

Reply:
The only concern I have with doing that is maintainability. Right now the 3 sql backends are *very* similar, so it's relatively easy to roll changes out across all 3 because it's largely a matter of copy+paste+tweak. The more differences creep in, the harder that's going to get long term. I'm happy to go ahead and make a special version of WriteAuthorizationModel for SQLite without the historical baggage if you think that's the best path forward.